### PR TITLE
MODKBEKBJ-171 API Tests: Packages Endpoint Updates for Tags

### PR DESCRIPTION
**Pull request by stanislav-mureiko**
## Purpose
Updated tests for https://issues.folio.org/browse/MODKBEKBJ-171

## Approach
Updated tests for packages

### CODE REVIEW

File: mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json [hunk 1]
```diff
@@ -7732,7 +7821,7 @@
 											"raw": ""
 										},
 										"url": {
-											"raw": "{{protocol}}://{{url}}:{{okapiport}}/eholdings/packages/{{packageId}}?include=",
+											"raw": "{{protocol}}://{{url}}:{{okapiport}}/eholdings/packages/{{packageId}}?include=prov",
```
Comment (natali-zaitseva) [inline mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json]: please remove `prov` value, seems it vas a small typo.

Comment (stanislav-mureiko) [reply mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json]: Thanks, @natali-zaitseva. Done.

File: mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json [hunk 2]
```diff
@@ -1033,6 +1033,16 @@
...
+									"//Check that tagList is not empty",
+									"pm.test('TagList is not empty', function(){",
+									"    pm.expect(response.data.attributes.tags.tagList.size) !== 0;",
+									"});",
+									"    ",
+									"//Check that tagList is not empty",
+									"pm.test('Creating tagList', function(){",
+									"    pm.expect(response.data.attributes.tags.tagList[0]).eq(\"foo-tag\");",
+									"});",
+									"    ",
```
Comment (carolegodfrey) [inline mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json]: Below this -- where we check for expected attributes (line 1054/1055) -- should we also check for tags to be complete?
      pm.expect(response.data.attributes).to.include.all.keys("contentType", "customCoverage", "isCustom", "isSelected", "name", "packageId", 
        "packageType", "providerId", "providerName", "selectedCount", "titleCount", "visibilityData", "allowKbToAddTitles");

Comment (carolegodfrey) [inline mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json]: Also makes sense to have checks on tags.tagList after we have confirmed that keys includes tags (similar to where the check of other content is done)

File: mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json [hunk 3]
```diff
@@ -6792,6 +6812,16 @@
...
 													"",
+													"//Check that tagList is not empty",
+													"pm.test('Creating tagList', function(){",
+													"    pm.expect(response.data.attributes.tags.tagList.size) !== 0;",
+													"});",
+													"    ",
+													"//Check that tagList is not empty",
+													"pm.test('Creating tagList', function(){",
+													"    pm.expect(response.data.attributes.tags.tagList[0]).eq(\"foo-tag\");",
+													"});",
```
Comment (carolegodfrey) [inline mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json]: Similar to above, check of taglist should be done after response.data has been confirmed and it has been confirmed that tags attribute is present (around line 6842)

File: mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json [hunk 4]
```diff
@@ -7961,6 +7991,22 @@
...
+															"pm.test('Creating tagList', function(){",
+															"    pm.expect(response.data.attributes.tags.tagList[0]).eq(\"foo-tag\");",
+															"    pm.expect(response.data.attributes.tags.tagList[1]).eq(\"another-tag\");",
+															"});",
+															"",
+															"//Check that tagList has 2 elements",
+															"pm.test('Creating tagList', function(){",
+															"    pm.expect(response.data.attributes.tags.tagList.size) === 2;",
+															"});",
+															"",
```
Comment (carolegodfrey) [inline mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json]: Same comment as above on this check happening after attributes check

Comment (stanislav-mureiko) [reply mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json]: Thanks @carolegodfrey . All issues have been fixed.

**Pull request merged**